### DISCUSSION TITLE
How can I use the filter function to filter on the obtained records when using https://plugin-search-pro.vuejs.press/guide.html

### DISCUSSION THREAD

**Discussion by joseaquino5** [vuepress-theme-hope/vuepress-theme-hope#3593] [Q&A] [answered]
Source discussion: https://github.com/vuepress-theme-hope/giscus-discussions/discussions/670

When implementing `vuepress-plugin-search-pro` I want to be able to filter the records obtained to be shown to the users, I don't know how I can address this. Per the docs https://plugin-search-pro.vuejs.press/guide.html#customize-search-options seems to be not possible:

![image](https://github.com/vuepress-theme-hope/giscus-discussions/assets/17714907/56771a2e-d044-483f-b457-ee1549542260)

Is there a workaround on this?

**Comment by joseaquino5** [comment]
Can someone please help me with this?

**Reply by Mister-Hope** [reply]
I am trying to find a graceful solution for this,

**Reply by joseaquino5** [reply]
So many thanks for looking at this, please let me know when a new release with this feature is out.

**Reply by Mister-Hope** [reply]
Sure!

Currently I am focusing a full rewrite of theme docs. And I closed some old issues which exisits for month.

This is more like a new feature request, so I will take sometime to introduce some new stuff about this.

**Reply by joseaquino5** [reply]
I understand, I really appreciate it  whatever effort you do on this, since this is a open source library whatever you can do in the time you have will be good, I will have an eye on this.

**Comment by Mister-Hope** [comment, marked as answer]
Filter is already supported now
